cosmwasm/lib/wasmi-runtime/Enclave_private.pem

```diff
@@ -0,0 +1,39 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Whoopsy doopsy

Reply:
```suggestion
-----BEGIN RSA PRIVATE KEY-----asdasd
```

Reply:
This is fine, it's just a test file from intel for now (google its content)

Reply:
Taken from here: https://github.com/intel/intel-sgx-ssl/blob/master/Linux/sgx/test_app/enclave/TestEnclave_private.pem

Reply:
If it's not doing anything we should remove this file from the PR

Reply:
@assafmo I know, jk :)

Reply:
It's used to sign the enclave so we do need this file: https://github.com/enigmampc/EnigmaBlockchain/blob/6cd448d376da812de940b7f20c870ad64f52c398/cosmwasm/lib/Makefile#L149

Reply:
But we'll want to use our own private key in production. This is fine for now.